### PR TITLE
feature: use keepalvied to add vip for virtual cluster

### LINKED ISSUE #635: support ha for tenant‘s VirtualCluster
**What would you like to be added**:

Currently, when creating a tenant's cluster, the services of the tenant's cluster cannot provide services to the outside world through a high-availability mode. Therefore, when creating a tenant cluster, kubenest also needs the ability to dynamically create keepalive plus haproxy or directly use metalLB to expose services externally.

**Why is this needed**:

### PR DESCRIPTION
**Pull request by village-way**
#### What type of PR is this?

/kind feature
#### Which issue(s) does this PR fix?

Fixes #635 
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/kubenest/tasks/data.go [hunk 1]
```diff
@@ -18,10 +18,12 @@ type InitData interface {
 	RemoteClient() clientset.Interface
 	KosmosClient() versioned.Interface
 	DataDir() string
+	VirtualClusterVersion() string
```
Comment (duanmengkk) [inline pkg/kubenest/tasks/data.go]: What is the usage of VirtualClusterVersion

Comment (village-way) [reply pkg/kubenest/tasks/data.go]: This line of code can be deleted, it should have been introduced when resolve code conflict.

**Comment by village-way** [comment]
close this pr with new pr #646

**Pull request closed without merging**